.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         node-version: 16
         cache: 'yarn'
     - run: yarn install
+    - run: ./packages/blob-reader/scripts/update-fixtures.sh
```

Comment:
For some reason I thought we run this script on CI already to ensure our unit tests run against the last dom, but turns out this wasn't the case. I was considering to add this as a `pretest` npm script, but then every local test run will update the fixtures as well which might be confusing to other contributors so I went with this approach.

Reply:
Would it be good to run this on a schedule, like once a week or so, and automatically PR the changes to keep the files in sync?

Reply:
Good idea. Is this something you want to look into?

Reply:
We were doing something very similar to this for [GitHub-Dark](https://github.com/StylishThemes/GitHub-Dark/blob/master/.github/workflows/regenerate.yml). A PR created using the `GITHUB_TOKEN` won't trigger workflows so a PAT or GitHub App will be needed, similar to the Dependabot auto-merge workflow.